gallery.html

```diff
@@ -169,9 +170,8 @@ <h3>Scalar data</h3>
                   </td>
                 </tr>
                 <tr>
-                  
                   <td >
-                    <h4>Dragon demo</h4>
+
```

Comment:
I believe you shifted all the titles by one. 
thanks for fixing that!

Reply:
basically the title for the "dragon demo" should be here and not below

Reply:
oh ! right ^^ I will fix that.